cmd/gzv/cli/gzv.go

```diff
@@ -185,13 +185,32 @@ func (gzv *Gzv) Run() {
 
 	clearCmd := app.Command("clear", "Clear the data of blockchain")
 
+	replayCmd := app.Command("replay", "replay the existing blocks")
+	srcDir := replayCmd.Flag("src", "directory of database for replaying").Required().String()
+	destDir := replayCmd.Flag("dest", "directory of database for storing the replayed data").String()
+
+	pruneCmd := app.Command("prune", "fully prune state data offline")
+	srcDB := pruneCmd.Flag("db", "database directory for pruning").Required().String()
+	srcSmallDB := pruneCmd.Flag("sdb", "small database directory for pruning which stores for the pruning mode").Default("").String()
```

Comment:
default value should set to d_small which as same as default value running in prune mode.

Reply:
should not have default value， because 
There is no small db in most scenarios